demo/index.html

```diff
@@ -86,6 +86,22 @@
         </form>
       </div>
 
+      <div class="panel">
+        <table>
+          <tr>
+            <th colspan="2">Custom Command Registration</th>
```

Comment:
After finishing  the other PR #7 (or in preparation for it) please title this panel: "Custom Interactive Command" and likewise change the titles of the other panels to: "Player Command" and "Custom Player Command".